TUNIC/TUNIC.asl

```diff
@@ -274,7 +274,7 @@ update
 
 start
 {
-	if (old.Scene == 2 && current.Scene == 24)
+	if (current.Scene == 25)
```

Comment:
Disapprove of this change. Please edit this back to a proper check for a change from one scene to another.

Reply:
Done, I've changed it to include the check again and updated the check on scene id from 80 -> 81 when setting current.Scene, which I had missed before.